app/consumer-democracy/app.go

```diff
@@ -506,6 +509,8 @@ func New(
 		distrtypes.ModuleName,
 		slashingtypes.ModuleName,
 		evidencetypes.ModuleName,
+		consumertypes.ModuleName, // Note: consumer beginblocker before staking module
```

Comment:
@jstr1121 Why is this order necessary?

Reply:
Beginblocker order shouldn't matter AFAIK